singularity-eos/eos/eos_davis.hpp

```diff
@@ -186,7 +327,7 @@ void DavisReactants::ValuesAtReferenceState(Real &rho, Real &temp, Real &sie, Re
 }
 
 #if 0
```

Comment:
This section of code is not compiled, should we remove it?

Reply:
As discussed, if we don't need this anymore for the PTE solver formulation (we don't) then it can go.

Reply:
:+1: